src/legacy/server/csp/index.ts

```diff
@@ -26,6 +26,7 @@ export const DEFAULT_CSP_RULES = Object.freeze([
   `script-src 'unsafe-eval' 'nonce-{nonce}'`,
   'worker-src blob:',
   'child-src blob:',
+  `style-src 'unsafe-inline' 'self'`,
```

Comment:
__question:__ is there any reason we don't want to use `nonce` for inline styles as well?

Reply:
Using the `nonce` has caused quite a few issues, as discussed [here](https://github.com/elastic/kibana/issues/42497) and I'd like to find a way to move toward `self` even for `script-src`. I could see the argument being made for using the `nonce` until we switch `script-src` to `self`, but it's quite a bit more work which we'd rather quickly get rid of.

Reply:
Okay, I see, thanks for the explanation. Yeah, let's see how it goes then.